.github/workflows/packages.yml

```
@@ -41,72 +41,75 @@
+        uses: ./.github/actions/add-label
+        with:
+          label: 'needs-yarn-install'
+
+      - name: Remove Label
+        if: ${{ steps.verify-yarn-lock.outputs.files_changed != 'true' && github.event_name != 'push' }}
+        uses: ./.github/actions/remove-label
+        with:
+          label: 'needs-yarn-install'
+
+      - name: List all changed files tracked and untracked files
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
```

actionlint

property "verify-changed-files" is not defined in object type {verify-yarn-lock: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)